lib/Tokenizer.js

```diff
@@ -635,6 +637,9 @@ Tokenizer.prototype.write = function(chunk){
 Tokenizer.prototype._parse = function(){
 	while(this._index < this._buffer.length && this._running){
 		var c = this._buffer.charAt(this._index);
+		// Detect Markdown code so that it is parsed as text instead of HTML
+		if (c === '`')
```

Comment:
No space before opening parentheses 😢

Let's be consistent with [L152](https://github.com/MarkBind/htmlparser2/pull/1/files#diff-00550ec11d6b5101df5a54c5fee7cc2eL152) of the file (braces even for single line of code).